.circleci/config.yml

```diff
@@ -828,8 +828,9 @@ jobs:
             pulumi config set --path unchained:common.eks.nodeGroups[0].instanceTypes[3] r6i.4xlarge
             pulumi config set --path unchained:common.eks.nodeGroups[0].instanceTypes[4] r6id.4xlarge
             pulumi config set --path unchained:common.eks.traefik.autoscaling.enabled true
-            pulumi config set --path unchained:common.eks.traefik.autoscaling.cpuThreshold 20
+            pulumi config set --path unchained:common.eks.traefik.autoscaling.memoryThreshold 50
```

Comment:
want to just add `cpuThreshold` as well so we can explicitly see the config and not have to know/remember defaults?